src/Umbraco.Configuration/ConfigsFactory.cs

```diff
@@ -25,10 +25,26 @@ public Configs Create()
             configs.Add<IUmbracoSettingsSection>("umbracoConfiguration/settings");
             configs.Add<IHealthChecks>("umbracoConfiguration/HealthChecks");
 
+            configs.Add<IUserPasswordConfiguration>(() => new DefaultUserPasswordConfig());
             configs.Add<ICoreDebug>(() => new CoreDebug());
             configs.Add<IConnectionStrings>(() => new ConnectionStrings());
             configs.AddCoreConfigs(_ioHelper);
             return configs;
         }
     }
+
+    // Default/static user password configs
+    // TODO: Make this configurable somewhere - we've removed membership providers, so could be a section in the umbracosettings.config file?
+    // keeping in mind that we will also be removing the members membership provider so there will be 2x the same/similar configuration
+    internal class DefaultUserPasswordConfig : IUserPasswordConfiguration
+    {
+        public int RequiredLength => 12;
```

Comment:
Any reason to increase this from 10 to 12?

Reply:
haha, nope, just thought it was 12 off the top of my head, we can change when we implement the actual config.